bitnami/fluentd/README.md

```diff
@@ -141,6 +144,20 @@ The following tables lists the configurable parameters of the fluentd chart and
 | `aggregator.service.loadBalancerSourceRanges`   | Addresses that are allowed when service is LoadBalancer                                                        | `[]`                                                                                                    |
 | `aggregator.service.clusterIP`                  | Static clusterIP or None for headless services                                                                 | `nil`                                                                                                   |
 | `aggregator.service.annotations`                | Annotations for the aggregator service                                                                         | `{}`                                                                                                    |
+| `aggregator.ingress.enabled`                    | Enable ingress controller resource                                                                             | `false`                                                                                                 |
+| `aggregator.ingress.certManager`                | Add annotations for cert-manager                                                                               | `false`                                                                                                 |
+| `aggregator.ingress.hostname`                   | Default host for the ingress resource                                                                          | `wordpress.local`                                                                                       |
+| `aggregator.ingress.path`                       | Default path for the ingress resource                                                                          | `/`                                                                                                     |
```

Comment:
Probably a typo, you forgot to add the parameter `aggregator.ingress.pathType` here.